packages/settings-view/lib/install-panel.js

```diff
@@ -86,7 +86,7 @@ export default class InstallPanel {
             <div className='text native-key-bindings' tabIndex='-1'>
               <span className='icon icon-question' />
               <span ref='publishedToText'>Packages are published to </span>
-              <a className='link' onclick={this.didClickOpenAtomIo.bind(this)}>atom.io</a>
+              <a className='link' onclick={this.didClickOpenAtomIo.bind(this)}>pulsar-edit.dev</a>
```

Comment:
```suggestion
              <a className='link' onclick={this.didClickOpenAtomIo.bind(this)}>web.pulsar-edit.dev</a>
```

Like mentioned by @Daeraxa should get this updated to the site where we can actually view packages